windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -184,7 +184,37 @@ UwbDeviceConnector::SessionInitialize(uint32_t sessionId, UwbSessionType session
 {
     std::promise<UwbStatus> resultPromise;
     auto resultFuture = resultPromise.get_future();
-    // TODO: invoke IOCTL_UWB_SESSION_INIT
+
+    wil::shared_hfile handleDriver;
+    auto hr = OpenDriverHandle(handleDriver, m_deviceName.c_str());
+    if (FAILED(hr)) {
+        PLOG_ERROR << "failed to obtain driver handle for " << m_deviceName << ", hr=" << hr;

```

Comment:
Let's make sure this follows the pre-existing output format for logging hrs (hex with base prefix):


```suggestion
        PLOG_ERROR << "failed to obtain driver handle for " << m_deviceName << ", hr=" << std::showbase << std::hex << hr;
```